ui/src/literature/components/LiteratureDate/LiteratureDate.jsx

```diff
@@ -1,23 +1,28 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import './LiteratureDate.scss';
+
 class LiteratureDate extends Component {
   render() {
-    const { date } = this.props;
+    const { date, wrapperClassName } = this.props;
 
     return (
-      <span>{date}</span>
+      <span className="__LiteratureDate__">
+        <span className={wrapperClassName}>{date}</span>
```

Comment:
this is confusing, and unnecessary to push down to this component just for displaying date with paranthesis. I think you can just wrap it with paranthesis in `jsx markup` wherever you want.

Reply:
it looked uglier to me doing this though but  👍